config.travis.py

```diff
@@ -57,3 +57,6 @@ class TRAVEL:
 }
 RESUME_BUCKET = "resumet"
 
+# Json webtoken
+JWT_SECRET = "1002C8E19420D4D5F117C13BC1971875F7CE4791ED0901A60E46549AAA74C80F"
```

Comment:
Make sure you are not using this same secret in Dev or prod. Kinda spooked me a bit seeing this. You can also use Travis envy vars to hide secrets like this

Reply:
Yeah no worries, wasn't planning on setting on the same secret in dev or prod. Was just doing a quick fix for travis at the time.